src/features/home/Home.tsx

```diff
@@ -29,11 +31,17 @@ const types = [
   {id: 3, type: TypesDish.Snacks},
 ];
 
+@inject(Stores.DishStore)
+@observer
 export class Home extends React.Component<Props, State> {
   public state: State = {
     currentType: TypesDish.Foods,
   };
 
+  public componentDidMount() {
+    this.props.dish.fetchDishes();
```

Comment:
@DiamondRock597 When we implementing async request we need to have a flag from mobx store and show activity indicator.